src/redux/reducers/experimentSettings/updateExperimentInfo.js

```diff
@@ -17,7 +17,7 @@ const updateExperimentInfo = produce((draft, action) => {
   draft.info.sampleIds = sampleIds;
 
   // Experiment id was updated so processing config requires reloading
-  draft.processing.meta.loading = true;
+  draft.processing = { meta: metaInitialState };
```

Comment:
Now to allow the action creator`loadProcessingConfig` to go through we check if `processingConfig` is loaded, so wiping it from the store is enough to have the same behavior as we had before in this case.